pkg/controller/node/nodecontroller.go

```diff
@@ -371,6 +402,42 @@ func (nc *NodeController) Run() {
 			}
 		}, nc.nodeMonitorPeriod, wait.NeverStop)
 
+		go wait.Until(func() {
+			nc.evictorLock.Lock()
```

Comment:
yup, but you can use UUID to keep namespace, or do you need for something else? I'd rather avoid parsing stuff...